oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnector.java

```diff
@@ -144,4 +147,4 @@ protected Schema getSchema(int sqlType, String typeName, int scale, int precisio
     }
     return super.getSchema(sqlType, typeName, scale, precision, columnName, isSigned, handleAsDecimal);
   }
-}
+}
```

Comment:
minor nit: we usually have newline char at the end of the file, hence we are seeing red circle here.